footer.php

```diff
@@ -68,7 +68,7 @@ class="button button--facebook">
 						class="button button--flickr">
 						<i class="fab fa-flickr"></i> Flickr
 					</a>
-					<a rel="noopener" href="https://vimeo.com/studieverenigingid"
+					<a rel="noopener" href="https://vimeo.com/studieverenigingid/videos"
```

Comment:
Dit zou trouwens normaal ook in een losse commit en pull request gaan, maar voor nu is het fine